kratos/factories/mapper_factory.h

```diff
@@ -157,6 +163,26 @@ class KRATOS_API(KRATOS_CORE) MapperFactory
     /// Default constructor.
     MapperFactory() {}
 
+    static std::string GetPath() {
+        if constexpr (TSparseSpace::IsDistributed()) {
+            return std::string("mappers.all.mpi");
+        }
+        return std::string("mappers.all");
```

Comment:
those names are only internally used in the registry right? I.e. nothing the user has to deal with?

Reply:
The users of MapperFactory should be able to continue interacting with it as usual. One could get the mappers directly form the registry using those paths, but this is not used at the moment.